tox.ini

```diff
@@ -56,14 +57,15 @@ commands=
     python -c "import eth_rlp"
 skip_install=true
 
-[testenv:py311-wheel-windows]
+[testenv:windows-wheel]
 deps=
     wheel
     build[virtualenv]
 allowlist_externals=
     bash.exe
 commands=
     python -m pip install --upgrade pip
+    python --version
```

Comment:
Nit: May not need this in the output, but if it is useful/required it could be moved so that it is the first command.

Reply:
Moved it up! I left that line and a couple in `config.yml` for future debugging purposes. `pyenv-win` or CircleCI may change how the aliases `python` or `python3` work in the future. I decided to still use the aliases to allow a lot less repetition in the setup though.